benchmarks/advection_reaction_3D/raja/CMakeLists.txt

```diff
@@ -58,6 +58,25 @@ if(BUILD_ARKODE AND BUILD_CVODE AND BUILD_IDA)
   install(FILES README.md ../scripts/compare_error.py ../scripts/compute_error.py ../scripts/pickle_solution_output.py
     DESTINATION "${BENCHMARKS_INSTALL_PATH}/advection_reaction_3D/raja")
 
+  set(BENCHMARK_VAR
+    "--method ARK-IMEX --nls tl-newton --tf 0.01 --dont-save\;arkimex_tlnewton_raja"
+    "--method ARK-DIRK --nls newton --tf 0.01 --dont-save\;arkdirk_newton_raja"
+    "--method CV-BDF --nls newton --tf 0.01 --dont-save\;cvbdf_newton_raja"
+    "--method IDA --nls newton --tf 0.01 --dont-save\;ida_newton_raja"
+    )
+
+  # NUM_CORES based on Lassen system for 1 node
+  foreach(benchmark_tuple ${BENCHMARK_VAR})
+    list(GET benchmark_tuple 0 benchmark_args)
+    list(GET benchmark_tuple 1 identifier)
+
+    sundials_add_benchmark(advection_reaction_3D_raja advection_reaction_3D_raja advection_reaction_3D
+      NUM_CORES 40
```

Comment:
Add a CMake option that is `SUNDIALS_BENCHMARKS_NUM_CORES` and use it instead (default value can be 40).